azext_iot/central/params.py

```diff
@@ -130,7 +130,14 @@ def load_central_arguments(self, _):
         context.argument("timeout", arg_type=event_timeout_type)
         context.argument("properties", arg_type=event_msg_prop_type)
 
-    with self.argument_context("iot central app validate-messages") as context:
+    with self.argument_context("iot central diagnostics monitor-events") as context:
+        context.argument("timeout", arg_type=event_timeout_type)
+        context.argument("properties", arg_type=event_msg_prop_type)
+        context.argument(
+            "module_id", options_list=["--module-id", "-m"], help="Iot Edge Module ID",
+        )
+
+    with self.argument_context("iot central diagnostics validate-messages") as context:
         context.argument("timeout", arg_type=event_timeout_type)
```

Comment:
Can we do a refactor of all the remaining params and nest them under the appropriate sub groups since the new subgroups are now decided?

Reply:
moving only unique params under groups.